evalml/tests/model_understanding_tests/test_graphs.py

```diff
@@ -478,6 +478,9 @@ def test_graph_confusion_matrix_default(X_y_binary, data_type, make_data_type):
     assert np.array_equal(heatmap['z'], conf_mat)
     assert np.array_equal(heatmap['customdata'], conf_mat_unnormalized)
     assert heatmap['hovertemplate'] == '<b>True</b>: %{y}<br><b>Predicted</b>: %{x}<br><b>Normalized Count</b>: %{z}<br><b>Raw Count</b>: %{customdata} <br><extra></extra>'
+    annotations = fig.__dict__['_layout_obj']['annotations']
+    for i in range(len(annotations)):
+        assert 'text' in annotations[i]
```

Comment:
What is this checking for? Why are we looking for the string `text`? :o

Reply:
@angela97lin This check makes sure the annotation text is in the figure. Before adding in the annotations, this check would've failed, but we are making sure that the annotations all have a text field here so that in the figure, there will be labels

Reply:
This is what annotations looks like:

```python
(layout.Annotation({
    'font': {'color': '#FFFFFF'}, 'showarrow': False, 'text': '1.000', 'x': 0, 'xref': 'x', 'y': 0, 'yref': 'y'
}), layout.Annotation({
    'font': {'color': '#000000'}, 'showarrow': False, 'text': '0.000', 'x': 1, 'xref': 'x', 'y': 0, 'yref': 'y'
}), layout.Annotation({
    'font': {'color': '#000000'}, 'showarrow': False, 'text': '0.480', 'x': 0, 'xref': 'x', 'y': 1, 'yref': 'y'
}), layout.Annotation({
    'font': {'color': '#FFFFFF'}, 'showarrow': False, 'text': '0.520', 'x': 1, 'xref': 'x', 'y': 1, 'yref': 'y'
}))
```

I guess checking `text` is present means that the confusion matrix heat map actually has values in each grid cell. I think a comment would be useful but otherwise looks good to me!